tests/test_multieurlex_utils.py

```diff
@@ -0,0 +1,102 @@
+import os
+from unittest.mock import patch
+
+import datasets
+import pyarrow as pa
+from datasets.formatting import PythonFormatter
+from datasets.formatting.formatting import LazyRow
+
+from arc_spice.data import multieurlex_utils
+
+# def extract_articles(
+#     item: LazyRow, languages: list[str]
+# ) -> dict[str, str] | dict[str, dict[str, str]]:
+
+TEST_ROOT = os.path.dirname(os.path.abspath(__file__))
+
+
+def _create_row(text) -> LazyRow:
+    pa_table = pa.Table.from_pydict({"text": [text]})
+    formatter = PythonFormatter(lazy=True)
+    return formatter.format_row(pa_table)
+
+
+def _create_multilang_row(texts_by_lang: dict[str, str]) -> LazyRow:
+    d = [{"text": texts_by_lang}]
+    pa_table = pa.Table.from_pylist(d)
+    formatter = PythonFormatter(lazy=True)
+    return formatter.format_row(pa_table)
+
+
+def test_extract_articles_single_lang():
+    langs = ["en"]
```

Comment:
Should we loop over all languages here?

Reply:
We only have test data for single lang for english (as the loader works a bit differently). I can create it for others if we want to expand the tests but it _should_ be the same functionality. Thoughts?

Reply:
ah ok is this addressed by the other comment re. testing all languages?